docroot/modules/custom/dct_configurations/templates/footerblock.html.twig

```diff
@@ -0,0 +1,4 @@
+<p class="footer--sponsor--intro">{{ "Drupal community hosting powered by" | t }}</p>
+<a href="javascript:void(0)" title="Acquia">
```

Comment:
Please do not use this function, this was used for front development only. You should set real links.